project/.travis.yml.twig

```diff
@@ -64,6 +63,8 @@ matrix:
 {% endfor %}
     - php: '{{ php|last }}'
       env: SYMFONY_DEPRECATIONS_HELPER=0
+    - php: hhvm
+      dist: trusty
   allow_failures:
     - php: nightly
     - php: hhvm
```

Comment:
Shouldn't this line be removed then? or atleast add `dist: trusty`.

But I am not sure about moving hhvm out of allow_failures. Can't we do this change on the `allow_failures` section?